detection_rules/attack.py

```diff
@@ -3,16 +3,24 @@
 # you may not use this file except in compliance with the Elastic License.
 
 """Mitre attack info."""
+import os
+import re
+import time
 
 import json
-import os
 import requests
 from collections import OrderedDict
 
 from .semver import Version
 from .utils import get_etc_path, get_etc_glob_path, read_gzip, gzip_compress
 
 PLATFORMS = ['Windows', 'macOS', 'Linux']
+CROSSWALK_FILE = get_etc_path('attack-crosswalk.json')
+SUB_TECHNIQUE_REDIRECT_MAP_FILE = get_etc_path('attack-replacement-map.json')
```

Comment:
added the crosswalk file, but the redirect map is just as (if not more) useful

Reply:
can you rename this file? maybe something like `attack-technique-redirects`. it'll make it a little bit easier to tell the difference between the two